.devcontainer/bin/dev-kafka-start

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COMPOSE_FILE="${MORPHEUS_ROOT}/.devcontainer/docker-compose.yml"
+
+docker compose -f $COMPOSE_FILE up -d zookeeper kafka
```

Comment:
Normally, I spin up 3 instances of kafka to test multi-replica modes. Is that possible from here?

Reply:
We can just update the compose file to spin up 3 instances instead of 1. We could get fancier than that and use one compose file per service (kafka1/zookeer / kafka3/zookeeper / triton) and just pick which one to use based on a cli option, but I see no reason to do that when we can just fire up three.

Reply:
Is that something you want done now, or something we can update later? You may want to give some instructions on how to test your scenario.

Reply:
We can update that later if needed.